src/pudl/transform/eia930.py

```diff
@@ -0,0 +1,224 @@
+"""Module to perform data cleaning functions on EIA930 data tables."""
+
+import pandas as pd
+from dagster import AssetOut, Output, multi_asset
+
+import pudl
+
+logger = pudl.logging_helpers.get_logger(__name__)
+
+
+@multi_asset(
+    outs={
+        "core_eia930__hourly_balancing_authority_net_generation": AssetOut(
+            io_manager_key="parquet_io_manager"
+        ),
+        "core_eia930__hourly_balancing_authority_demand": AssetOut(
+            io_manager_key="parquet_io_manager"
```

Comment:
Note that right now, all the hourly EIA-930 data is getting written to Parquet only, because otherwise it takes 20 minutes to write the tables out, and they take up 8GB of space in SQLite, while the Parquet outputs take 90 seconds to write out and 260MB on disk.

However, this means that we've got empty tables inside the DB with the table schema (which is weird) and also that we will need to specifically pull the parquet outputs up out of the `parquet` folder for distribution.

Reply:
oh my i didn't really conceive of the fact that not exporting these tables to the db mean we don't get the db checks. that seems like a real problem actually. are these any schema checks done with the `parquet_io_manager`?

Reply:
All the same schema checks do get done regardless of whether it's written out to Parquet or SQLite. The only thing that I think we'll miss out on is FK validity, but the association tables which are built from the data tables here DO have FK relationships (for the BA codes), and they're being satisfied so I think that's okay.

Reply:
Verify that the pandera asset checks require joint uniqueness of the PK tuple.